app/lib/actions/pulp3/orchestration/content_view_version/import.rb

```diff
@@ -0,0 +1,41 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module ContentViewVersion
+        class Import < Actions::EntryAction
+          def plan(content_view_version, path:)
+            action_subject(content_view_version)
+            sequence do
+              smart_proxy = SmartProxy.pulp_primary!
+              importer_output = plan_action(::Actions::Pulp3::ContentViewVersion::CreateImporter,
+                                           content_view_version_id: content_view_version.id,
+                                           smart_proxy_id: smart_proxy.id,
+                                           path: path).output
+
+              import_output = plan_action(::Actions::Pulp3::ContentViewVersion::Import,
+                                           content_view_version_id: content_view_version.id,
+                                           smart_proxy_id: smart_proxy.id,
+                                           importer_data: importer_output[:importer_data],
+                                           path: path).output
+
+              plan_action(Actions::Pulp3::Repository::SaveVersions, content_view_version.importable_repositories.pluck(:id),
+                          tasks: import_output[:pulp_tasks])
+
+              plan_action(::Actions::Pulp3::ContentViewVersion::DestroyImporter,
+                            smart_proxy_id: smart_proxy.id,
+                            importer_data: importer_output[:importer_data])
+            end
+          end
+
+          def humanized_name
+            _("Import")
```

Comment:
Would be nice to see something more descriptive on the task list, like "Import content view version"

Reply:
This comment still stands

Reply:
Oh I see it's been changed in the Katello action.  I guess that's what shows up in Foreman Tasks?